package.json

```diff
@@ -41,7 +44,7 @@
     "babel-preset-env": "^1.6.1",
     "danger": "^3.0.5",
     "gulp": "^3.9.1",
-    "npm-scripts-info": "^0.3.6"
```

Comment:
`npm-scripts-info` is still in the `yarn.lock` file, have you checked in the latest changes to that file?